app/components/ak-table/index.ts

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+
+interface AkTableSignature {
+  Element: HTMLDivElement;
+  Args: {
+    headerColor?: string;
+    variant?: string;
+    borderColor?: string;
+    hoverable?: boolean;
+    dense?: boolean;
+  };
+
+  Blocks: {
+    // FIXME: Add a better type for yielded block
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    default: any;
```

Comment:
They do but I was encountering some issues using their provided types. It was taking much time to fix so I had to pause it temporarily. This would be revisited. I would require you help when that time comes.